src/mmw/js/src/analyze/views.js

```diff
@@ -215,14 +187,6 @@ var TabPanelsView = Marionette.CollectionView.extend({
     },
     childView: TabPanelView,
 
-    events: {
-        'shown.bs.tab li a ': 'triggerBarChartRefresh'
-    },
-
-    triggerBarChartRefresh: function() {
-        $('#analyze-output-wrapper .bar-chart').trigger('bar-chart:refresh');
```

Comment:
I think this was necessary when we had the chart and table in the bottom bar. We needed to trigger this so that the bar chart would resize itself after the size of the bottom bar changed, I think.